securedrop/journalist_templates/base.html

```diff
@@ -21,7 +21,7 @@
     {% if g.user %}
       {% if g.show_v2_onion_eol_warning %}
       <div id="v2-onion-eol" class="warning-banner">
-        {{ gettext('<strong>Update Required:</strong> Your SecureDrop servers are still running v2 Onion Services, which are being phased out for security reasons. In February 2021, v2 Onion Services will be disabled, and your SecureDrop servers may become unreachable. <a href="//securedrop.org/v2-onion-eol" rel="noreferrer">Learn More</a>') }}
+        {{ gettext('<strong>Update Required:</strong>Your SecureDrop servers are running onion services v2, which will be deprecated February 2021. Your SecureDrop servers may become unreachable. <a href="//securedrop.org/v2-onion-eol" rel="noreferrer">Learn More</a>') }}
```

Comment:
```suggestion
        {{ gettext('<strong>Update Required:</strong>Your SecureDrop servers are running v2 onion services, which will be deprecated for security reasons come February 2021. Your SecureDrop servers may become unreachable. <a href="//securedrop.org/v2-onion-eol" rel="noreferrer">Learn More</a>') }}
```

Reply:
That is still not an accurate use of the term "deprecated". Unless you feel really strongly about the phrase "phased out" or some other aspect of the original string, I would suggest we close this PR @comradekingu, I can open a separate one for changing the spelling to the lower case variant in cases where `onion service` is not used in headlines.

Reply:
How so? Thought it was better. "Taken out of service" isn't true, because they will still work upstream.
https://en.wikipedia.org/wiki/Deprecation
https://www.collinsdictionary.com/dictionary/english/deprecate

I only feel strongly about the quality of the original string, and how I don't want to see it again.
If the other possible meaning of "phased out" is left without the qualifying date, one would not have to reiterate how it will stop working in a new sentence. (?)

"Your SecureDrop servers may become unreachable when v2 onion services are deprecated for security reasons come February 2021." is a better sentence in my head.

Reading a lengthy passage to try to best translate is more taxing than trying to understand it, and a recipe for translations made with no oversight, by different people, or with the use of direct translations. SD is exactly that.

Reply:
See my explanation here: https://github.com/freedomofpress/securedrop/pull/5375#issuecomment-658921716

The term "deprecation" is used for the period during which we recommend disabling, not for the removal itself. It's the latter (removal) which will occur in February 2021. This is also well-explained in the Wikipedia article you link to.

Reply:
> Edit: "Upgrade your SecureDrop servers to more secure v3 onion services so you can reach them after February 2021." is what I would go with.

This language won't work with the current implementation, because the same warning is displayed regardless of whether v3 services are enabled or not, _if_ v2 services are enabled. See the logic here: https://github.com/freedomofpress/securedrop/pull/5366/files#diff-4c2318c651fa1f217d6b6ed48e77e201R176

I do think it's worth considering a shorter warning if folks don't feel it's too later for that, so thanks for the suggestions -- will discuss at our team standup tomorrow. For now I've opened #5379 to standardize `onion service` usage throughout the codebase.

Reply:
"Your SecureDrop server still supports deprecated v2 onion services, Make sure it has v3 ones so that it will be reachable come February 2021." ?
or
"Make sure your SecureDrop server will be reachable after 2021 by making sure it supports the safer v3 onion services, and not just v2 which will stop working." ?